secrets-provider-test-chart/templates/job.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Values.global.namespace }}
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ .Values.rbac.serviceAccount.name }}
+      containers:
+      - image: {{ .Values.secretsProvider.image }}
+        imagePullPolicy: Never
+        name: {{ .Values.secretsProvider.name }}
+        env:
+        - name: SECRETS_DESTINATION
+          value: {{ .Values.secretsProviderEnvironment.secretsDestination | quote }}
+
+        # TODO, remove requirement in code
```

Comment:
Remove this TODO.
We did not plan to do it, so better keep the code as is.

Reply:
This means this will need to be in manifest which is no good. We should have 
```
- name: CONTAINER_MODE
  value: init
```
in our manifests if we aren't running as an init

Reply:
added task: https://app.zenhub.com/workspaces/palmtree-5d99d900491c060001c85cba/issues/cyberark/secrets-provider-for-k8s/166